src/BaseFirestoreRepository.spec.ts

```diff
@@ -31,6 +31,36 @@ describe('BaseRepository', () => {
     bandRepository = new BandRepository('bands');
   });
 
+  describe('limit', () => {
+    it('must limit the documents in a collection', async () => {
+      const twoBands = await bandRepository.limit(2);
```

Comment:
Following [this comment](http://) limit won't execute the query, so the right way to use limit

`const twoBands = await bandRepository.limit(2).find();`